sway-lsp/src/server.rs

```diff
@@ -794,16 +795,16 @@ mod tests {
             json!({
                 "contents": {
                     "kind": "markdown",
-                    "value": "```sway\nstruct Data\n```"
+                    "value": " Struct holding:\n\n 1. A `value` of type `NumberOrString`\n 2. An `address` of type `u64`"
```

Comment:
Is this extra space intentional?

```suggestion
                    "value": "Struct holding:\n\n 1. A `value` of type `NumberOrString`\n 2. An `address` of type `u64`"
```

Reply:
That space is there because it's reading from this doc comment
`/// Struct holding:` 

If the doc comment had of been written like `///Struct holding:` then there would be no white space at the beginning here. 

I guess we could have a discussion how we want to treat this, i.e do we want to autostrip white space so it's all identical? I'll leave that for another PR though.